tests/extensions/bleskomat/test_lnurl_api.py

```diff
@@ -126,11 +127,12 @@ async def test_bleskomat_lnurl_api_action_success(client, lnurl):
     )
     wallet = await get_wallet(bleskomat.wallet)
     assert wallet.balance_msat == 100000
-    WALLET.pay_invoice.reset_mock()
-    response = await client.get(f"/bleskomat/u?k1={secret}&pr={pr}")
-    assert response.json() == {"status": "OK"}
-    wallet = await get_wallet(bleskomat.wallet)
-    assert wallet.balance_msat == 50000
-    bleskomat_lnurl = await get_bleskomat_lnurl(secret)
-    assert bleskomat_lnurl.has_uses_remaining() == False
-    WALLET.pay_invoice.assert_called_once_with(pr, 2000)
+    if not is_regtest:
+        WALLET.pay_invoice.reset_mock()
```

Comment:
Why is the whole block in the `if`? Can't this test be run if it's regtest? Should be enough to just `reset_mock()` and go with it. 

Also, this (inside a test case) is not the place to reset the mock. It should be done in `conftest.py`.

Reply:
there is no pay_invoice mock in regtest, so it will fail.